drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -107,27 +107,43 @@
 #define APB4_FREQ	((AHB_FREQ)/(CONFIG_CLOCK_STM32_D3PPRE))
 
 /* Datasheet maximum frequency definitions */
-#define SYSCLK_FREQ_MAX	480000000UL
-#define AHB_FREQ_MAX	240000000UL
-#define APBx_FREQ_MAX	120000000UL
+#if defined(CONFIG_SOC_STM32H743XX) ||\
+    defined(CONFIG_SOC_STM32H745XX) ||\
+    defined(CONFIG_SOC_STM32H747XX) ||\
+    defined(CONFIG_SOC_STM32H750XX)
+	/* All h7 SoC with maximum 480MHz SYSCLK */
+	#define SYSCLK_FREQ_MAX		480000000UL
+	#define AHB_FREQ_MAX		240000000UL
+	#define APBx_FREQ_MAX		120000000UL
+#elif defined(CONFIG_SOC_STM32H723XX)
+	/* All h7 SoC with maximum 550MHz SYSCLK */
+	#define SYSCLK_FREQ_MAX		550000000UL
+	#define AHB_FREQ_MAX		275000000UL
+	#define APBx_FREQ_MAX		137500000UL
+#else
+	/* Default: All h7 SoC with maximum 280MHz SYSCLK */
+	#define SYSCLK_FREQ_MAX		280000000UL
+	#define AHB_FREQ_MAX		140000000UL
+	#define APBx_FREQ_MAX		70000000UL
```

Comment:
I'm not quite fond about indentation of #define's but I don't think there is a coding rule for that, so that's ok.

Reply:
I think to keep consistancy with the following defines enclosed by #if/#endif, the indentation should be removed.

Reply:
Removed